test/parallel/test-buffer-alloc.js

```diff
@@ -1,15 +1,11 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
-var Buffer = require('buffer').Buffer;
+const Buffer = require('buffer').Buffer;
+const SlowBuffer = require('buffer').SlowBuffer;
 
-// counter to ensure unique value is always copied
-var cntr = 0;
-
-var b = Buffer.allocUnsafe(1024);
-
-console.log('b.length == %d', b.length);
+const b = Buffer.allocUnsafe(1024);
 assert.strictEqual(1024, b.length);
```

Comment:
There are *many* places in the tests where they are in the wrong order. For now, I'm inclined to leave these as they are. It would be a good first contribution for someone tho :-)